authors/settings.py

```diff
@@ -51,9 +51,11 @@
     'authors.apps.authentication',
     'authors.apps.core',
     'authors.apps.profiles',
+    "authors.apps.report",
 
     'social_django',
-    "authors.apps.article"
+    "authors.apps.article",
+
 
 ]
```

Comment:
I Have seen you only have one endpoint for creating the report what about retrieving the reports, how can we retrieve them??

Reply:
This was not part of my task. Kindly read my description and pt story description.